docker/ubuntu/koandroid/Dockerfile

```diff
@@ -1,14 +1,16 @@
 FROM frenzie/kobase-python:0.1.0
 
 USER root
-RUN apt-get update
-RUN apt-get install -y --no-install-recommends openjdk-8-jdk ant
-RUN apt-get clean
-RUN rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
+RUN add-apt-repository -y ppa:openjdk-r/ppa && \
```

Comment:
What's the advantage of using this PPA?

Reply:
@houqp I'll merge this without this particular change for the moment because this is a regular package available in Ubuntu 16.04 and 18.04.